<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Maze</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #0C090A;
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MazeScene" name="mazeScene" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 1,
											  stop:0 #CC0099, stop:1 #FF9933);
	color: #FFFFFF;
	border-style: solid;
	border-color: #FFFFFF;
	border-width: 0px;
	font-size: 20px;
}

QPushButton::hover {
	border-width: 4px;
}

QPushButton::pressed {
	border-width: 0px;
	background-color:  #0C090A;
	color:  qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 1,
											  stop:0 #CC0099, stop:1 #FF9933);
}</string>
    </property>
    <property name="text">
     <string>GENERATE</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rowsSpinBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>521</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
	color: #FFFFFF;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="colsSpinBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>561</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
	color: #FFFFFF;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="widthLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #FFFFFF;
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>WIDTH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="heightLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #FFFFFF;
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>HEIGHT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #FFFFFF;
	font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>FILENAME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButtonMaze">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 1,
											  stop:0 #CC0099, stop:1 #FF9933);
	color: #FFFFFF;
	border-style: solid;
	border-color: #FFFFFF;
	border-width: 0px;
	font-size: 20px;
}

QPushButton::hover {
	border-width: 4px;
}

QPushButton::pressed {
	border-width: 0px;
	background-color:  #0C090A;
	color:  qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 1,
											  stop:0 #CC0099, stop:1 #FF9933);
}</string>
    </property>
    <property name="text">
     <string>LOAD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="caveModeButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 1,
											  stop:0 #CC0099, stop:1 #FF9933);
	color: #FFFFFF;
	border-style: solid;
	border-color: #FFFFFF;
	border-width: 0px;
	font-size: 20px;
}

QPushButton::hover {
	border-width: 4px;
}

QPushButton::pressed {
	border-width: 0px;
	background-color:  #0C090A;
	color:  qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 1,
											  stop:0 #CC0099, stop:1 #FF9933);
}</string>
    </property>
    <property name="text">
     <string>CAVE MODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButtonMaze">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 1,
											  stop:0 #CC0099, stop:1 #FF9933);
	color: #FFFFFF;
	border-style: solid;
	border-color: #FFFFFF;
	border-width: 0px;
	font-size: 20px;
}

QPushButton::hover {
	border-width: 4px;
}

QPushButton::pressed {
	border-width: 0px;
	background-color:  #0C090A;
	color:  qlineargradient(spread: pad, x1: 0, y1: 0, x2: 1, y2: 1,
											  stop:0 #CC0099, stop:1 #FF9933);
}</string>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MazeScene</class>
   <extends>QWidget</extends>
   <header>view/maze_scene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
